### ISSUE TITLE
better organization of the generated files

### ISSUE THREAD

**Issue by zaibon** [Jumpscale/go-raml#213]
Currently all the generated files are stores on the same directory.
In the case of a bug raml file, this can become a bit messy. Can we group all the json schema in a dedicated directory to make things a bit more organized ?

**Comment by iwanbk** [comment]
```
Can we group all the json schema in a dedicated directory to make things a bit more organized ?
```
Yep, it is a very good idea.

implemented in 42b8ec3a67aacef4bc0c9bbd885d9066cb888ea7

**Comment by zaibon** [comment]
thanks
